### PR TITLE
Fix #1512 Add video block to Block Kit support

### LINKED ISSUE #1512: Add video block to Block Kit support
"video" type blocks are now available in Block Kit.

>Showcase your cool cat videos in a new [video block](https://api.slack.com/reference/block-kit/blocks#video) with [Block Kit](https://api.slack.com/block-kit). Read on to learn more on [how to embed videos in your apps](https://api.slack.com/reference/block-kit/blocks#video).

* https://api.slack.com/changelog
* https://api.slack.com/reference/block-kit/blocks#video

#### Packages:

Select all that apply:

- [ ] `@slack/web-api`
- [ ] `@slack/rtm-api`
- [ ] `@slack/webhooks`
- [ ] `@slack/oauth`
- [ ] `@slack/socket-mode`
- [x] `@slack/types`
- [ ] I don't know

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

This pull request resolves #1512 

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/types/src/index.ts [hunk 1]
```diff
@@ -470,3 +470,16 @@ export interface CallUserExternal {
   display_name: string;
   avatar_url: string;
 }
+
+export interface VideoBlock extends Block {
+  type: 'video';
+  video_url: string;
+  thumbnail_url: string;
+  alt_text: string;
+  title: PlainTextElement;
```
Comment (seratch) [inline packages/types/src/index.ts:479]: verified the behavior -- these four are required while the rest are optional

**Review by seratch — COMMENTED** [review]
comment for reviewers

**Review by filmaj — APPROVED** [review]
LGTM! I guess unlike the python PR you sent, we can't do the string max length validation in this project?

**Comment by seratch** [comment]
@filmaj The reason why we maintain the validations on the SDK side in Python is mainly for consistency with the existing code. We don't need to add the ones to Node and Java.

**Pull request merged**